### PR TITLE
You can now buy double barrel shotgun from the goodie section

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About The Pull Request
For the absurd price of 1,800 credits, and a weapon permit, the double barrel shotgun can be yours!
## Why It's Good For The Game
Well you can buy the detective revolver, you can buy the combat shotgun(with armoury access) you can buy a lot of the weapons in general actually with enough money and access. I don't see why not the shotgun
## Changelog
:cl:
add: The Bartender's double barrel shotgun can be purchased for 1,800 credits on goodie tab
/:cl:

### CODE REVIEW

**Comment by DaCoolBoss** [comment]
My gut feeling is that this should be a subtype of the shotgun so it doesn't interact with traitor objectives, but the detective gun doesn't do this and doesn't cause problems so maybe it's not a big deal.

**Comment by CCC23** [comment]
finally bag shotgun

**Comment by projectkepler-ru** [comment]
> finally bag shotgun

I mean, you can bag the riot shotgun if it's sawn off so I don't see the problem? and most ammo arent available regardless so the effectiveness of the shotgun is only as good as you are knowledgable about how to use the shotgun

**Comment by jughu** [comment]
remember the whole reason why comapct shotgun isnt able to go on suit storage cause it was overused and automatic this is 2 shot automatic

**Comment by projectkepler-ru** [comment]
> remember the whole reason why comapct shotgun isnt able to go on suit storage cause it was overused and automatic this is 2 shot automatic

the- compact shotgun always went into the suit slot, infact it even fit in the backpack in older build

**Comment by necromanceranne** [comment]
I think they mean the combat shotgun, the full-sized shotgun. The compact version that the Head of Security gets can fit into suit storage.

I don't really foresee people chasing this. It's very rare to see people actually go out of their way to get the ,38 even if they normally have a weapon permit, like security. As a goodie, this is only really feasibly purchased by the crew member who can produce the most amount of money without much effort. The bartender, who already has one until he needs a replacement, so this PR serves its purpose with helping the roles have a chance of getting back lost equipment for their job (at their expense).

Maybe people will want sawnoffs for some reason, but I think if people wanted a baggable gun they would probably be buying the revolver.

**Comment by SimplySmiley123** [comment]
> Maybe people will want sawnoffs for some reason, but I think if people wanted a baggable gun they would probably be buying the revolver.

Doubt this as shotguns fell off from grace.
They're a novelty now if anything.

**Comment by projectkepler-ru** [comment]
> I think they mean the combat shotgun, the full-sized shotgun. The compact version that the Head of Security gets can fit into suit storage.
> 
> I don't really foresee people chasing this. It's very rare to see people actually go out of their way to get the ,38 even if they normally have a weapon permit, like security. As a goodie, this is only really feasibly purchased by the crew member who can produce the most amount of personal money without much effort. The bartender, who already has one until he needs a replacement, so this PR serves its purpose with helping the roles have a chance of getting back lost equipment for their job (at their expense).
> 
> Maybe people will want sawnoffs for some reason, but I think if people wanted a baggable gun they would probably be buying the revolver.

I specifically kept in mind that lethal munition is much harder to source now and the only use I could think to come of this is if you used shotgun dart on it for a two shot RDX-Teslium combo. Or something like that.

File: code/modules/cargo/goodies.dm [hunk 1]
```diff
@@ -308,3 +308,10 @@
 	desc = "A less cheap imported climbing hook. Absolutely no use outside of planetary stations."
 	cost = PAYCHECK_CREW * 5
 	contains = list(/obj/item/climbing_hook)
+
+/datum/supply_pack/goody/double_barrel
+	name = "Double Barrel Shotgun"
+	desc = "Lost your beloved bunny to a demonic invasion? clown broke in and stole your beloved gun? no worries! get a new gun so long as you can pay the absurd fees."
```
Comment (JohnFulpWillard) [inline code/modules/cargo/goodies.dm:314]:
```suggestion
	desc = "Lost your beloved bunny to a demonic invasion? Clown broke in and stole your beloved gun? No worries! Get a new gun so long as you can pay the absurd fees."
```

**Comment by projectkepler-ru** [comment]
ty @JohnFulpWillard

**Pull request merged**